mica/archive/aca_dark/__init__.py

```diff
@@ -1,2 +1,2 @@
 from dark_cal import *
-from dark_model import *
+from chandra_aca.dark_model import *
```

Comment:
Should this import be replaced with some code that will issue a deprecation warning if the user comes looking for dark_model in mica?